<?xml version="1.0"?>
<launch>
    <arg name="dir_name" default="$(find ros_m5bala_drive)/config/qr/" />
    <arg name="file_name" default="" />
    <arg name="sequence_no" default="-1" />

    <node pkg="ros_m5bala_drive" type="image_publisher.py" name="publish_image_for_ros_m5bala" output="screen">
        <param name="dir_name" value="$(arg dir_name)" />
        <param name="file_name" value="$(arg file_name)" />
        <param name="sequence_no" value="$(arg sequence_no)" />
    </node>

</launch>
